Todo-Redux/src/js/modules/prelogin/actions.js

```diff
@@ -0,0 +1,32 @@
+const uid = () => Math.random().toString(34).slice(2);
+
+export function addTodo(text) {
+  return {
+    type: 'ADD_TODO',
+    payload: {
+      id: uid(),
+      status: false,
+      text: text
+    }
+  };
+}
+
+export function searchTodo(text) {
+  return {
+    type: 'SEARCH_TODO',
+    payload: text
+  }
+}
+
+export function searchEmpty() {
+  return {
+    type: 'SEARCH_EMPTY'
+  }
+}
+
+export function displayTodo(id) {
+  return {
+    type: 'DISPLAY_TODO',
+    payload: id
+  }
```

Comment:
Please create separate constant file and keep 'SEARCH_EMPTY" 'DISPLAY_TODO' means all the action types over there.  Keep all those constant in one object and export that object

Reply:
Created constants.js file